idseq_dag/steps/run_star.py

```diff
@@ -202,7 +202,7 @@ def sync_pairs(fastq_files, max_discrepancies=0):
                 fqf=fastq_files,
                 example=(outstanding_r0 or outstanding_r1).popitem()[0])
             log.write(msg)
-            assert discrepancies_count <= max_discrepancies, msg
```

Comment:
Suggestion: raising the threshold instead of removing this line

Reply:
Agree! This was just intended as a test to debug.

Reply:
Which brings us to the question of what's a good value for the threshold.

Let's assume these bugs are distributed uniformly.   We already do uniform subsampling later to just 2 million reads.   So a threshold up to (total number of reads - 2 million) would actually not affect ultimate result quality very much.

Of course there are quite a few filtering steps before the subsampling.  It would be nice to look at existing data for the typical cull rate in that filtering and CDR, then go back from 2 million to however many reads we should want to have after STAR in order to have 2 million before sampling, and that would give us the perfect threshold.